miner/miner.go

```diff
@@ -287,10 +290,44 @@ func (m *Miner) hasPower(ctx context.Context, addr address.Address, ts *types.Ti
 	return !power.MinerPower.Equals(types.NewInt(0)), nil
 }
 
+// Note: copied from the chainstore method. Maybe we should expose this over the api?
```

Comment:
plsdo

Reply:
So, we can add a separate method for just this, which feels weird, given that its doing something you can easily do *over* the api, or we can add it to the MinerGetBaseInfo method, since thats called around here anyways, thoughts?

Reply:
Yeah, MinerGetBaseInfo is a decent place for it